Documentation/spelling_wordlist.txt

```diff
@@ -2,6 +2,10 @@ AArch
 AKS
 Alemayhu
 Alibaba
+Anthos
+Automount
+BIGTCP
+BPF
```

Comment:
we probably only need autoamount in this commit.

Reply:
Thanks! Fixed in the latest push.